packages/itwin/tree-widget/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@itwin/tree-widget-react",
-  "version": "2.0.2",
+  "version": "2.2.0",
```

Comment:
The version is incorrect. It should be 2.1.0. Did you run `beachball sync` and then `beachball bump`? It should have been enough to run `beachball bump` . Other package also have incorrect version

Reply:
thx for catching that, updated

Reply:
Looks good. I think you can by pass checks and merge.